app/views/albums/_album.json.jbuilder

```diff
@@ -0,0 +1,2 @@
+json.extract! album, :id, :name, :created_at, :updated_at
```

Comment:
Consider using https://github.com/stas/jsonapi.rb serializer for your API json responses